hasura/graphql-engine
# misc console actions fixes

### Description


- Move setting process.hrtime to webpack so that codegen is usable in
pro
- Fix warnings and unnecessary console errors
- Disable snippets for aceeditor

### Affected components



### Steps to test and verify



Create an action and go to its codegen tab. Should work without any errors in dev console.
